### PR TITLE
[ONNX] dort to inline onnx model before running ort

### PR DESCRIPTION
**Pull request by BowenBao**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #110313
* #110477
* #110178
* __->__ #108376

### CODE REVIEW

File: torch/onnx/_internal/onnxruntime.py [hunk 1]
```diff
@@ -797,6 +798,18 @@ def maybe_map_to_meta_val(value):
                 opset_version=self._resolved_onnx_exporter_options.onnx_registry.opset_version,
             )
 
+            # Inline model before executing in ORT.
+            # Temporary solution to improve performance until runtimes adapt to function format.
```
Comment (thiagocrepaldi) [inline torch/onnx/_internal/onnxruntime.py:802]: Adding a TODO comment with the issue URL would help the future us to track and remove this workaround when the time is right

**Review by thiagocrepaldi — COMMENTED** [review]
Adding a unit test to ensure the inlining is happening would also help track behavior change

**Review by thiagocrepaldi — COMMENTED** [review]
@BowenBao is it still relevant for us?

**Comment by BowenBao** [comment]
> @BowenBao is it still relevant for us?

not anymore, closing

**Pull request closed without merging**